.gitignore

```diff
@@ -1,2 +1,3 @@
 /hubble
 install/kubernetes/hubble.yaml
+.idea/
```

Comment:
This doesn't hurt to add, but this belongs in your global .gitignore, really.

Reply:
fair enough. reverted.